rmd_files/RNAseq_report.Rmd

```diff
@@ -4225,7 +4225,7 @@ Out of the `r if ( runFusionChunk ) { nrow(fusions) } else { c("0") }` fusion ev
 
 **`r if ( !runFusionChunk ) { c("No fusion genes were detected!") }`**
 
-```{r fusions_summary_table, comment = NA, message=FALSE, warning=FALSE}
+```{r fusions_summary_table_Arriba, comment = NA, message=FALSE, warning=FALSE, eval = runArribaChunk}
```

Comment:
I think we should use "runPizzlyChunk" rather than "runArribaChunk" as fusions results do not depend solely on Arriba output, some results can derive from Pizzly and so the chunk should be run